src/component/RoadmapCanvas.js

```diff
@@ -34,35 +35,34 @@ export class RoadmapCanvas {
    * @param {Roadmap} roadmap
    */
   render(roadmap) {
-    if (this.type === "story") {
-      this._render(roadmap, roadmap.getSortedStoryList());
-    } else if (this.type === "assignee") {
-      this._render(roadmap, roadmap.getSortedAssigneeList());
+    // TODO: check. Is this condition need?
```

Comment:
I think, Roadmappy should not supports this feature natively.

Roadmappy user has other way to solve this problem.

```js
let roadmap1 = new Roadmappy({ ..., type: 'story' });
roadmap1.render();
let roadmap2 = new Roadmappy({ ..., type: 'assignee' });
roadmap2.render();
```